playwright-e2e/pages/osteo/consent-page.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, Locator, Page } from '@playwright/test';
+import { AngioPageBase } from 'pages/angio/angio-page-base';
```

Comment:
do we need to use AngioPageBase here?

Reply:
Will remove this and some other unused imports!